### PR TITLE
Match params of init script with the ones of systemd unit

### PR DESCRIPTION
**Pull request by tooooots**
#### Pull Request (PR) description

Adds the same ulimit settings in the init script as the ones that are already present in systemd unit file.

#### This Pull Request (PR) fixes the following issues

Improve #226

### CODE REVIEW

File: templates/init.erb [hunk 1]
```diff
@@ -50,9 +50,17 @@ if [ -f /etc/default/kafka ]; then
 fi
 
 start() {
-  ulimit -n 65536
+
+  <% unless @limit_nofile.nil? -%>
```
Comment (bastelfreak) [inline templates/init.erb]: Can't we use this?
```suggestion
  <% if @limit_nofile -%>
```

**Comment by bastelfreak** [comment]
I'm afraid we somehow need to get https://github.com/voxpupuli/puppet-kafka/pull/278 and https://github.com/voxpupuli/puppet-kafka/pull/275 merged before we can proceed here :(

**Comment by bastelfreak** [comment]
Thanks @tooooots. Can you also add a unit test for the change?

**Comment by tooooots** [comment]
thanks for the quick feedbacks @bastelfreak

**Comment by bastelfreak** [comment]
Can you have a look at the failing travis jobs?

**Comment by dhoppe** [comment]
@tooooots Could you please do a rebase against the ```master``` branch? Since ```modulesync``` has been updated, the ```.travis.yml``` is out of date. For example it will do the RSpec tests against Ruby 2.1.9, which seems to break Travis CI.

**Pull request merged**